app/app/src/main/kotlin/com/hedvig/android/app/ui/HedvigAppState.kt

```diff
@@ -208,6 +209,10 @@ private fun NavigationTrackingSideEffect(navController: NavController) {
           }
         }
       }
+      GlobalRum.get().startView(
+        key = destination,
+        name = destination.route ?: destination.displayName,
```

Comment:
`destination.displayName` seems to have `@get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` on it, which means that they optimally want to restrict the callers only to the library itself.

Basically they'd want it to have the `internal` modifier but they can't for some reason.
So the tl;dr is that we probably should avoid calling it, as it does not have any guarantees that it will still be there forever, or that the implementation won't change (so our metrics might start reporting changes even though nothing we did changed) and so on.

I think we can do something like `destination.route ?: "Unknown route",` instead, since I don't thin there's realistically ways that we'll be getting a null route anyway. And if we do we can see if it is a real problem or not. For our destinations, none of it should be null anyway.

Reply:
Good catch, fixing